docs/ref/models/querysets.txt

```diff
@@ -2024,31 +2024,55 @@ ignored since these two optimizations do not make sense together.
 Depending on your database backend capabilities, the results will either be
 loaded all at once or streamed from the database using server-side cursors.
 
-Using server-side cursors
-^^^^^^^^^^^^^^^^^^^^^^^^^
+**Using server-side cursors**
```

Comment:
The second commit is changing some things that could be done in the first commit. I think keeping the use of headings is okay but I guess some styling is needed for `<h5>`.